frontend/src/pages/BlogPreviewPage.tsx

```
@@ -0,0 +1,69 @@
+                <p className="text-sm text-muted-foreground leading-relaxed">{blog.subtitle}</p>
+              )}
+            </div>
+            {imageUrl && (
+              <div className="h-64 w-full overflow-hidden rounded-lg bg-muted">
+                <img src={imageUrl} alt={blog.title} className="h-full w-full object-cover" />
+              </div>
+            )}
+          </div>
+          <div
+            className="text-sm leading-relaxed text-foreground space-y-4"
+            dangerouslySetInnerHTML={{ __html: blog.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)